code/app/src/main/res/layout/activity_main.xml

```diff
@@ -18,4 +18,10 @@
         android:layout_height="wrap_content"
         android:text="Get Local Notification" />
 
+    <Button
+        android:id="@+id/btn_triggerIAM"
+        android:layout_width="190dp"
```

Comment:
Any reason to hard code the width, can we use wrap content and add some padding?